### ISSUE TITLE
Cherrytree 1.03: Shift + Arrow key no longer marks text, but moves nodes in tree

### ISSUE THREAD

**Issue by mhaeusser** [giuspen/cherrytree#2400]
**Version, Operative system**
cherrytree_1.0.3.0_win64_setup.exe

**Describe the bug**
In an editor window, when I hold shift and then press the arrow up key, this does not, as in previous versions and other editors, mark the text, but surprisingly moves the currently selected node in the navigation tree on the left.

**To Reproduce**
see above

**Comment by giuspen** [comment]
Look at https://github.com/giuspen/cherrytree/issues/2397 for temporary workaround

**Comment by firepainting** [comment]
Just reporting that this happens to me too. Linux Mint 21.2 Cinnamon. 

The above workaround works. Shift+Arrow now moves a node only after the node is clicked. The old shortcut still works (Shift+Alt+Arrow).

Thank you.

**Comment by firepainting** [comment]
Just to be clear I only removed the shortcuts `Shift+Up/Down/Right/Left Arrow` for `Move the Selected Node Up/Down/Right/Left`.

**Comment by giuspen** [comment]
Thanks @firepainting I'm going to release 1.0.4 including this fix this weekend. If you want to try it before and report the test results, I am happy to make a Debian package for you though.

**Comment by giuspen** [comment]
(or an AppImage)

**Comment by firepainting** [comment]
Thanks @giuspen. I'm not really a tester-tester, but if you can make a Debian package, then I'll be using it asap and report to you anything I may find. At least I'll be using it every day until this weekend. If that sounds good to you then I'd love that!

**Comment by giuspen** [comment]
Perfect thanks I'll make a deb package in a moment and paste/report here when ready

**Comment by giuspen** [comment]
Please try https://www.giuspen.net/software/cherrytree_1.0.3-3+10~Mint21.2_amd64.deb

**Comment by firepainting** [comment]
Many thanks! However, I'm getting these:

![gui_error](https://github.com/giuspen/cherrytree/assets/50393421/8a9edcfa-3174-4e62-8e31-931b0df80c7e)

```
~$ sudo dpkg -i /home/username/Downloads/cherrytree_1.0.3-3+10~Mint21.2_amd64.deb
dpkg-deb: error: '/home/username/Downloads/cherrytree_1.0.3-3+10~Mint21.2_amd64.deb' is not a Debian format archive
dpkg: error processing archive /home/username/Downloads/cherrytree_1.0.3-3+10~Mint21.2_amd64.deb (--install):
 dpkg-deb --control subprocess returned error exit status 2
Errors were encountered while processing:
 /home/username/Downloads/cherrytree_1.0.3-3+10~Mint21.2_amd64.deb
```

**Comment by giuspen** [comment]
Sorry I'll double check later and get back

**Comment by giuspen** [comment]
I downloaded from the same location and it did install correctly
```
giuspen@mint2204:/tmp$ sudo dpkg -i cherrytree_1.0.3-3+10~Mint21.2_amd64.deb 
[sudo] password for giuspen:            
Selecting previously unselected package cherrytree.
(Reading database ... 407431 files and directories currently installed.)
Preparing to unpack cherrytree_1.0.3-3+10~Mint21.2_amd64.deb ...
Unpacking cherrytree (1.0.3-3) ...
Setting up cherrytree (1.0.3-3) ...
Processing triggers for desktop-file-utils (0.26+mint3+victoria) ...
Processing triggers for gnome-menus (3.36.0-1ubuntu3) ...
Processing triggers for mailcap (3.70+nmu1ubuntu1) ...
Processing triggers for hicolor-icon-theme (0.17-2) ...
Processing triggers for man-db (2.10.2-1) ...
```
Maybe try to download again?
I'll try also to attach here.
The sha256sum:
```
giuspen@mint2204:/tmp$ sha256sum cherrytree_1.0.3-3+10~Mint21.2_amd64.deb 
1e8e5547fe92fe044b44e42594641c6e82a53cb0eb355528013567d8f1bb937c  cherrytree_1.0.3-3+10~Mint21.2_amd64.deb
```

**Comment by giuspen** [comment]
[test-cherrytree.tar.gz](https://github.com/giuspen/cherrytree/files/13675488/test-cherrytree.tar.gz)

**Comment by firepainting** [comment]
> [test-cherrytree.tar.gz](https://github.com/giuspen/cherrytree/files/13675488/test-cherrytree.tar.gz)

^^ This worked! Now installed and in use. Turns out this is 2.9MB; the first link was only 1.3kB.

Will report, if any. If I'm silent then that probably means all is good ;) (OP, sorry if I hijacked this thread!)

One more thing, I scanned those issues concerning shortcut changes in favor of Windows users (on the latest Release Notes). In general I could adapt to such changes but I wish shortcut changes could be avoided all in all. Old habits are so convenient :)) This isn't a huge deal to me, it's more of a teeny inconvenience. I understand that stuff should work for everyone so with regard to shortcuts I hope that gets sorted out!

Second thing, just a massive thanks to you for this wonderful app and the support!

**Comment by giuspen** [comment]
Many thanks @firepainting pre release test is very useful.
With the shortcuts it's difficult to please everyone but the most important thing is that the defaults are not causing any trouble to nobody. Then you can configure your preferred once in the preferences dialog and export your settings from File--Preferences--Export that should satisfy all cases.
Note that in the new version that you are testing, the overridden keyboard shortcuts are emphasized in bold and can be reset individually

**Comment by mhaeusser** [comment]
> Will report, if any. If I'm silent then that probably means all is good ;) (OP, sorry if I hijacked this thread!)
Never mind. But I did not expect so many e-mails...

**Comment by fabriziobertocci** [comment]
Giuseppe, 
I can confirm your .deb package install correctly and fixes the problem of the shift+cursors.

One suggestion: please don't change the shortcuts or allow people that are used with the old shortcuts to continue using them.
For example the shortcut for bullet list has changed from CTRL+ALT+3 to Shift+Alt+L (?!?!)
BTW if you need any help with the project, let me know. I have been using it for many years now and I found it such a wonderful tool. Thank you for the great effort!

**Comment by giuspen** [comment]
Thanks @fabriziobertocci 
I understand the frustration with the shortcuts, I'm sorry about that, I've been trying to adjust to a default that is not causing trouble to any platform and ended up annoying many users.
Note though that you can adjust your keyboard shortcuts in the preferences dialog, tab keyboard shortcuts, and export your preferences for later re-import.
The Ctrl+Alt+... are causing a lot of pain to windows users on non English keyboards, I had to get rid of all those from the defaults, but you can apply such shortcuts in your preferences.
Any help would certainly be appreciated Fabrizio, coding or also just documenting. If you have some spare time and would like to try you can drop me an email you can find it in help--about dialog

**Comment by giuspen** [comment]
Fixed in v1.0.4

**Comment by tititototutu** [comment]
Thanks a lot for your diligence and a lot more for cherrytree! :)
